enterprise/views.py

```diff
@@ -105,6 +107,18 @@ def get_global_context(request):
         'LMS_ROOT_URL': settings.LMS_ROOT_URL,
         'platform_name': platform_name,
         'header_logo_alt_text': _('{platform_name} home page').format(platform_name=platform_name),
+        'welcome_text': constants.WELCOME_TEXT.format(platform_name=platform_name),
+        'enterprise_welcome_text': constants.ENTERPRISE_WELCOME_TEXT.format(
+            enterprise_customer_name=enterprise_customer.name,
+            platform_name=platform_name,
+            strong_start='<strong>',
+            strong_end='</strong>',
+            line_break='<br/>',
+            privacy_policy_link_start="<a href='{pp_url}' target='_blank'>".format(
+                pp_url=settings.MKTG_URLS.get('PRIVACY', 'https://www.edx.org/edx-privacy-policy'),
```

Comment:
Can we use [`branding.api.get_privacy_url()`](https://github.com/edx/edx-platform/blob/2c3ff72bf19e5aa02345e10854785aeb0214a4de/lms/djangoapps/branding/api.py#L492) here instead?  This lets us use Site Configuration or `MKTG_URLS`, depending on what is configured.

Reply:
Ah yeah that'd be better. In fact it made me remember that we're using [`get_configuration_value`](https://github.com/edx/edx-enterprise/blob/master/enterprise/utils.py#L601), which encapsulates the use of a similar function in the platform: https://github.com/edx/edx-platform/blob/2c3ff72bf19e5aa02345e10854785aeb0214a4de/openedx/core/djangoapps/site_configuration/helpers.py#L98-L131 -- it just looks in site configuration, or even microsite if that fails, and ultimately returns the default, which we set to be this settings thing.

Since I don't want to introduce a new use of something like the branding API in edx-enterprise, I'll use `get_configuration_value` if that's alright.

Reply:
Actually you're right, we will have to do this because `PRIVACY` is just a path, not a full URI.

Reply:
Cool, thank you!